content/geometry/introduction.adoc

```diff
@@ -48,10 +48,19 @@ Users can continue using their familiar tools and processes to manage geometry,
-Individual objects, such as vehicles, SHALL be linked using the https://publications.pages.asam.net/standards/ASAM_OpenSCENARIO/ASAM_OpenSCENARIO_XML/latest/07_components_scenario/07_02_storyboard_entities.html#_3d_models_for_entities[model3d property].
-* ASAM Open Simulation Interface (OSI) is a specification for interfaces between models and components in a distributed simulation. OSI primarily focuses on the environmental perception of automated driving functions, but it also covers interfaces for models of traffic participants.
-The 3D environment SHALL be linked in OSI using the https://opensimulationinterface.github.io/osi-antora-generator/asamosi/current/gen/structosi3_1_1GroundTruth.html#a83042861723a4a9e890a53aa98d88858[GroundTruth::model_reference].
-Individual objects,like vehicles SHALL be linked with the https://opensimulationinterface.github.io/osi-antora-generator/asamosi/current/gen/structosi3_1_1MovingObject.html#a07558573bee7a5fa2f0729e1cad1325f[MovingObject::model_reference] or the https://opensimulationinterface.github.io/osi-antora-generator/asamosi/current/gen/structosi3_1_1StationaryObject.html#aad1c24fcdb11699954bf3494b8632288[StationaryObject::model_reference].
+* ASAM OpenDRIVE standardizes the description of road networks, including geometric definitions of the road, but primarily focuses on the semantic description of the road. 
+  The ASAM OpenMATERIAL object class "Environment" enhances this by allowing the definition of a comprehensive 3D environment that aligns with an ASAM OpenDRIVE map.
+* ASAM OpenSCENARIO is a file format created to describe dynamic content in driving and traffic simulators, with a primary focus on complex maneuvers involving multiple entities like vehicles and pedestrians.
+  ASAM OpenMATERIAL defines node structures and metadata for these object classes to seamlessly integrate ASAM OpenSCENARIO definitions with a 3D model.
+  The asset file (.xoma) of a static 3D environment SHALL be linked in ASAM OpenSCENARIO using the https://publications.pages.asam.net/standards/ASAM_OpenSCENARIO/ASAM_OpenSCENARIO_XML/latest/generated/content/RoadNetwork.html[SceneGraphFile property].
+  Asset files (.xoma) of individual objects, such as vehicles, SHALL be linked using the https://publications.pages.asam.net/standards/ASAM_OpenSCENARIO/ASAM_OpenSCENARIO_XML/latest/07_components_scenario/07_02_storyboard_entities.html#_3d_models_for_entities[model3d property].
+  Both environment and objects may either be given as an absolute path or as a path relative to the OpenSCENARIO file (.xosc).
+  A relative path is indicated by a dot (.) as the first character.
+* ASAM Open Simulation Interface (OSI) is a specification for interfaces between models and components in a distributed simulation.
```

Comment:
I added the note as suggested with one difference: I changed the SHALL to a MAY in the last sentence, because OSI does not necessarily have to be used.
If you approve, you can directly merge.